### ISSUE TITLE
gephi: garbled text

### ISSUE THREAD

**Issue by armijnhemel** [NixOS/nixpkgs#25639]
## Issue description

Text in Gephi is garbled
![gephi](https://cloud.githubusercontent.com/assets/10587952/25845910/13311858-34b0-11e7-9f67-c0ccc1d1b5df.png)

This is on my Fedora 25 system. I do notice the following warnings when starting Gephi:

libEGL warning: DRI2: failed to open i965 (search paths /run/opengl-driver/lib/dri)
libEGL warning: DRI2: failed to open i965 (search paths /run/opengl-driver/lib/dri)
libEGL warning: DRI2: failed to open swrast (search paths /run/opengl-driver/lib/dri)
libGL error: unable to load driver: i965_dri.so
libGL error: driver pointer missing
libGL error: failed to load driver: i965
libGL error: unable to load driver: i965_dri.so
libGL error: driver pointer missing
libGL error: failed to load driver: i965
libGL error: unable to load driver: swrast_dri.so
libGL error: failed to load driver: swrast
libEGL warning: DRI2: failed to open i965 (search paths /run/opengl-driver/lib/dri)
libEGL warning: DRI2: failed to open i965 (search paths /run/opengl-driver/lib/dri)
libEGL warning: DRI2: failed to open swrast (search paths /run/opengl-driver/lib/dri)

### Steps to reproduce

1. install Fedora 25
2. build gephi
3. run gephi

## Technical details

LSB Version:	:core-4.1-amd64:core-4.1-noarch:cxx-4.1-amd64:cxx-4.1-noarch:desktop-4.1-amd64:desktop-4.1-noarch:languages-4.1-amd64:languages-4.1-noarch:printing-4.1-amd64:printing-4.1-noarch
Distributor ID:	Fedora
Description:	Fedora release 25 (Twenty Five)
Release:	25
Codename:	TwentyFive

**Comment by lheckemann** [comment]
As far as I understand, it shouldn't be searching /run/opengl-driver on non-NixOS distributions and instead using the host's OpenGL drivers. As for why this is happening and how to fix it, I unfortunately have no clue.

**Comment by armijnhemel** [comment]
This seems reasonable, as I am also noticing it in other places. Perhaps @edolstra has a clue?

**Comment by georgewsinger** [comment]
I'm getting the `libEGL warning: DRI2: failed to open swrast (search paths /run/opengl-driver/lib/dri)` on Arch Linux for a custom built package. Setting a symlink `/run/opengl-driver/lib/dri -> /usr/lib/dri` so that `swrast.so` can be found doesn't help.

**Comment by knedlsepp** [comment]
I had a similar problem on openSUSE that could be solved with a font anti-aliasing setting, as described here in the case of Ubuntu: https://askubuntu.com/questions/584922/how-do-i-fix-fonts-not-rendering-and-missing-letters

**Comment by Mic92** [comment]
I fonts looks ugly I also recommend to set `_JAVA_OPTIONS` as follow in .zshrc/.bashrc:
```
export _JAVA_OPTIONS='-Dawt.useSystemAAFontSettings=on -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel'
```

**Comment by Mic92** [comment]
If the latest pull request, fixes your problem, I will backport it to `18.03` as well.
